modules/configuring-your-ide.adoc

```diff
@@ -0,0 +1,171 @@
+:_module-type: PROCEDURE
+
+[id="configuring-your-ide_{context}"]
+= Configuring your IDE
+
+[role='_abstract']
+You can configure some notebook workbenches to get the most out of your data science work.
+
+== Configuring Code Server
+Incorporating standard database clients into your Code Server environment can streamline your workflow, offering a seamless interface for managing databases. You can incorporate different Database Management System (DBMS) clients into your Code Server workbench through built-in extensions or by creating a custom notebook image using a Dockerfile. 
```

Comment:
There is a discussion on this PR opendatahub-io/notebooks!390 whether we shall use `Code Server` or whether we should rather follow what `code-server`.

Reply:
Updated to code-server in all instances.